docker/sink-compose1.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
```

Comment:
Instead of copying `sink-compose.yml` into multiple files, can we reuse it and make required changes through env. vars or command line args? So anything that needs to be configurable, (e.g., port numbers, service names, etc.) is provided through config parameters and we only have one copy of the compose file.

Reply:
This is a nice suggestion, I have made these changes in the latest commits. Please have a look.